conda/recipes/libraft/meta.yaml

```diff
@@ -99,12 +104,6 @@ outputs:
         - cuda-profiler-api {{ cuda_profiler_api_host_version }}
         - libcublas {{ libcublas_host_version }}
         - libcublas-dev {{ libcublas_host_version }}
-        - libcurand {{ libcurand_host_version }}
```

Comment:
To confirm, the compiled parts of the library don’t use libcurand, libcusolver, or libcusparse?

Note that because this package has a dependency on libraft-headers, the build will pull those math library packages anyway because they are run dependencies of libraft-headers — but they will be loosely pinned at 11.x “run versions” at build time rather than 11.8 “host versions”. It might make sense to keep these definitions so that you have tighter build-time pinnings, since the libs have to be fetched anyway.